airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -249,34 +249,27 @@
           }
         ]
       },
-      "use_1s1t_format": {
```

Comment:
Now that we're removing this, but we're still using this for Postgres, MySQL, etc and this code is now shared in the JDBC base classes - won't we need a new mechanism to ensure that the default value of `False` isn't returned?

Reply:
This PR removes `use_1s1t_format` and `TypingAndDedupingFlag.isDestinationV2()` for Redshift only (namely `spec.json` and `RedshiftStagingS3Destination.java`). There are still several references to `TypingAndDedupingFlag.isDestinationV2()` in generic JDBC code in the CDK that Postgres and MySQL will use. Postgres, etc. will just need to add the spec option in their config like the other DW's have done in the past.

Does that make sense or did I miss something?

Reply:
Oh I understand your question now, let me think about this. I thought we had this problem for BQ/Snowflake too.

Reply:
(BigQuery/Snowflake incident I was thinking of: #31717)

What do you think of keeping the spec option but with `airbyte_hidden: true` and `default: true` until both Postgres and MySQL GA and we fully remove usage of `TypingAndDedupingFlag.isDestinationV2()` in the JDBC code path?

Thanks for noticing this btw.

Reply:
update: need to solve this a different way bc the value won't be true after upgrading V1 connections who currently have a value of false